src/main/java/com/playtika/homework2/task1/model/listOfDevices/Computer.java

```diff
@@ -0,0 +1,14 @@
+package com.playtika.homework2.task1.model.listOfDevices;
```

Comment:
move `Device` and its inheritors to `package com.playtika.homework2.task1.model`